src/scss/components/_weather.scss

```diff
@@ -0,0 +1,32 @@
+.weather-today-nav {
+  @include media-breakpoint-down(md) {
+    border-bottom: 1px solid $gray-lighter;
+    border-top: 1px solid $gray-lighter;
+  }
+
+  .weather-date {
+    color: $gray-dark;
+    font-size: .8rem;
+  }
+
+  .weather-icon {
+    font-size: 1.75rem;
+
+    @media (max-width: 979px) {
```

Comment:
Breakpoint variable reference would be preferable here, along with a mobile-first approach.

e.g.

```
.weather-icon {
  font-size: 2.75rem;

  @include media-breakpoint-up(lg) {
    font-size: 1.75rem;
  }
}
```

Reply:
Fixed in d1ba289.